content/en/docs/reference/glossary/external-control-plane.md

```diff
@@ -0,0 +1,10 @@
+---
+title: External Control Plane
+test: n/a
+---
+
+An external control plane is a [control plane](/docs/reference/glossary/#control-plane)
+that externally manages mesh workloads running in their own [clusters](/docs/reference/glossary/#cluster)
+or other infrastructure. The control plane may, itself, be deployed in a cluster, although not
+in one of the [primary clusters](/docs/reference/glossary/#primary-cluster) of the mesh it's controlling.
```

Comment:
I don't think this sentence is correct. There is nothing about a `primary cluster` that implies it has to be part of the `mesh`. It's just a cluster that runs a control plane.

Reply:
I don't think the sentence is implying a primary cluster has to be part of the mesh. I intentionally avoided the issue by not calling the control plane's cluster anything. I'm just saying the control plane's cluster cannot be "one of the primary clusters in the mesh". Maybe slightly different wording would be better?
```suggestion
in one of the clusters that is part of the mesh it's controlling.
```

Reply:
Ah I think I misread it, my mistake. Your suggestion LG.